scripts/quarantine_zephyr.yaml

```diff
@@ -135,3 +135,19 @@
     - nrf5340dk_nrf5340_cpuapp_ns
     - nrf9160dk_nrf9160_ns
   comment: "Won't be fixed - https://nordicsemi.atlassian.net/browse/NCSDK-22771"
+
+- scenarios:
```

Comment:
just a nit: this is added in a section "won't fix" of the yaml (used just for ease of human verification). It has no effect on the functionality. No point in cleaning this now and wasting CI time

Reply:
another push will be needed, so I will fix this thx